src/tests/WebnnTest.cpp

```diff
@@ -0,0 +1,40 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "tests/WebnnTest.h"
+
+static WebnnTestEnvironment* gTestEnv = nullptr;
+
+void InitWebnnEnd2EndTestEnvironment() {
+    gTestEnv = new WebnnTestEnvironment();
+    testing::AddGlobalTestEnvironment(gTestEnv);
+}
+
+static void ErrorCallback(WebnnErrorType type, char const* message, void* userdata) {
+    if (type != WebnnErrorType_NoError) {
```

Comment:
This should probably cause a test failure too (unless in an EXPECT_CONTEXT_ERROR macro)

Reply:
it should, fixed in the new commit https://github.com/webmachinelearning/webnn-native/pull/1/commits/5483049d6aa356b72961587e84eff20ac95e1f2f